### PR TITLE
Release notes for 0.16.1

### PR DESCRIPTION
**Pull request by max-sixty**
Please make suggestions for any changes!

### CODE REVIEW

File: doc/whats-new.rst [hunk 1]
```diff
@@ -16,58 +16,70 @@ What's New
...
+
+Many thanks to the 36 contributors who contributed to this release:
+
+Aaron Spring, Akio Taniguchi, Aleksandar Jelenak, Alexandre Poux,
+Caleb, Dan Nowacki, Deepak Cherian, Gerardo Rivera, Jacob Tomlinson, James A.
+Bednar, Joe Hamman, Julia Kent, Kai Mühlbauer, Keisuke Fujii, Mathias Hauser,
+Maximilian Roos, Nick R. Papior, Pascal Bourgault, Peter Hausamann, Romain
+Martinez, Russell Manser, Samnan Rahee, Sander, Spencer Clark, Stephan Hoyer,
+Thomas Zilio, Tobias Kölling, Tom Augspurger, alexamici, crusaderky, darikg,
+inakleinbottle, jenssss, johnomotani, keewis, rpgoldman Breaking changes
```
Comment (dcherian) [inline doc/whats-new.rst:35]:
```suggestion
inakleinbottle, jenssss, johnomotani, keewis, rpgoldman 

Breaking changes
```

File: doc/whats-new.rst [hunk 2]
```diff
@@ -16,58 +16,70 @@ What's New
...
+
```
Comment (dcherian) [inline doc/whats-new.rst:36]:
```suggestion
```

**Review by dcherian — APPROVED** [review]
Thanks @max-sixty

File: doc/whats-new.rst [hunk 3]
```diff
@@ -16,58 +16,70 @@ What's New
...
-  - all versions of dask and dask.distributed released in the last 12 months (but no
-    older than 2.9)
-  - all versions of other packages released in the last 12 months
-
-  All are  up from 6 months (:issue:`4295`)
-  `Guido Imperiale <https://github.com/crusaderky>`_.
 - Use a wrapped array's ``_repr_inline_`` method to construct the collapsed ``repr``
   of :py:class:`DataArray` and :py:class:`Dataset` objects and
   document the new method in :doc:`internals`. (:pull:`4248`).
   By `Justus Magin <https://github.com/keewis>`_.
```
Comment (keewis) [inline doc/whats-new.rst:72]: I had meant to add the note to `whats-new.rst` for #4237 when adding support for `fillna` / `where` but I have yet to figure out to do so.
```suggestion
  By `Justus Magin <https://github.com/keewis>`_.
- Allow per-variable fill values in most functions. (:pull:`4237`)
  By `Justus Magin <https://github.com/keewis>`_.
```

File: doc/whats-new.rst [hunk 4]
```diff
@@ -16,58 +16,73 @@ What's New
...
-- Add support for parsing datetime strings formatted following the default
-  string representation of cftime objects, i.e. YYYY-MM-DD hh:mm:ss, in
-  partial datetime string indexing, as well as :py:meth:`~xarray.cftime_range`
-  (:issue:`4337`). By `Spencer Clark <https://github.com/spencerkclark>`_.
-- :py:meth:`~xarray.DataArray.to_dataframe` and :py:meth:`~xarray.Dataset.to_dataframe`
-  now accept a ``dim_order`` parameter allowing to specify the resulting dataframe's
-  dimensions order (:issue:`4331`, :pull:`4333`).
-  By `Thomas Zilio <https://github.com/thomas-z>`_.
+- Allow per-variable fill values in most functions. (:pull:`4237`)
```
Comment (max-sixty) [inline doc/whats-new.rst:74]:
```suggestion
- Allow per-variable fill values in most functions. (:pull:`4237`).
```

**Comment by max-sixty** [comment]
Does anyone know why RTD is failing? RTD is often inscrutable to me...

**Comment by keewis** [comment]
the relevant output of `sphinx` is:
```
/home/docs/checkouts/readthedocs.org/user_builds/xray/checkouts/4435/doc/whats-new.rst:55: WARNING: Bullet list ends without a blank line; unexpected unindent.
/home/docs/checkouts/readthedocs.org/user_builds/xray/checkouts/4435/doc/whats-new.rst:57: WARNING: Definition list ends without a blank line; unexpected unindent.
```

**Comment by max-sixty** [comment]
Thanks @keewis . I was looking at the bottom of the page

**Comment by keewis** [comment]
for warnings that don't print a stacktrace (sphinx warnings) you typically have to look at the output between `waiting for workers...` (directly below the `reading sources...` lines) and `pickling environment`

**Pull request merged**